components/SiteNavbar.vue

```diff
@@ -59,183 +83,33 @@
-            mt-1
-            px-3
-            py-0
-            text-gray-600
-            lg:py-2 lg:my-0
-            hover:bg-gray-100 hover:text-gray-800
-          "
-          href="#"
-          >Home</a
-        >
-        <a
-          class="
-            rounded
-            px-3
-            py-0
-            text-gray-600
-            lg:py-2
-            hover:bg-gray-100 hover:text-gray-800
-          "
-          href="#"
-          >Games</a
-        >
-        <a
-          class="
-            rounded
-            px-3
-            py-0
-            lg:py-2
-            text-gray-600
-            hover:bg-gray-100 hover:text-gray-800
-          "
-          href="#"
-          >About</a
-        >
-        <div
-          class="
-            relative
-            flex flex-col
-            w-full
-            px-2
-            lg:flex-row lg:w-auto lg:my-0
-          "
-        >
-          <input
-            class="
-              border border-gray-400
-              rounded
-              bg-white
-              px-4
-              py-2
-              pr-12
-              focus:outline-none
-            "
-            name="search"
-            placeholder="Search..."
-          />
-          <button type="submit" class="absolute right-6 bottom-3">
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              class="h-5 w-5"
-              fill="none"
-              viewBox="0 0 24 24"
-              stroke="currentColor"
-            >
-              <path
-                stroke-linecap="round"
-                stroke-linejoin="round"
-                stroke-width="2"
-                d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
-              />
-            </svg>
-          </button>
-        </div>
-      </div>
-      <div
-        class="flex flex-col w-full text-center lg:flex-row lg:w-auto lg:my-0"
-        v-show="mobileNavIsActive || lg"
-      >
-        <!-- Only visible on desktops screen sizes -->
-
-        <a
-          class="
-            border border-gray-300
-            rounded
-            px-3
-            py-2
-            m-1
-            text-gray-600
-            hidden
-            lg:inline lg:my-0
-            hover:text-gray-800 hover:bg-gray-100
-          "
-          href="#"
-          >Log In</a
-        >
-        <a
-          class="
-            bg-gray-600
-            rounded
-            px-3
-            py-2
-            m-1
-            hidden
-            text-gray-100
-            lg:inline lg:my-0
-            hover:text-white hover:bg-gray-900
-          "
-          href="#"
-          >Sign Up</a
-        >
-      </div>
     </div>
-  </nav>
+  </div>
 </template>
 
 <script>
 export default {
   data() {
     return {
-      mobileNavIsActive: false,
       lg: false,
-    }
+      mobileNavIsActive: false,
+    };
+  },
+
+  /* Need to disable the `sort-keys` rule here, since it
+   * conflicts with the `vue/order-in-components` rule.
```

Comment:
If we do want to disable one of them, I'd be in favour of disabling the `vue/order-in-components` as well, since I'd rather have all the keys ordered.